_data/changelogs/docs-settings.yml

```diff
@@ -2,6 +2,14 @@ title: Settings
 type: utility
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added new `$theme-utility-breakpoints-custom` setting.
+    summaryAdditional:  Users can now set custom breakpoints which will generate utilities alongside the default system breakpoints.
```

Comment:
**suggestion**:  Some minor tweaks to improve readability
```suggestion
    summaryAdditional:  Users can use this new setting to generate responsive variants of USWDS utilities at custom breakpoints. 
```

Reply:
@amyleadem I'd argue that the original is a little clearer. First, it mentions that this is a new setting and second it tells you that new utils can be generated from it.